crates/sui-indexer/src/indexer_reader.rs

```diff
@@ -1616,7 +1618,7 @@ impl IndexerReader {
         ))
     }
 
-    pub fn package_resolver(&self) -> Arc<Resolver<impl PackageStore>> {
+    pub fn package_resolver(&self) -> PackageResolver {
```

Comment:
It's necessary to fully specify this type, so the Indexer's system package task knows that it contains a cache that needs to be evicted.